packages/fargate-task/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "pulumi-fargate-task",
```

Comment:
A little nitpicky, but the package `pulumi-fargate-task` name could be confused with something like a [Fargate task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html).

Maybe something like `run-task-in-fargate`?